operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml-patch

```diff
@@ -13,3 +13,4 @@
       - manila.csi.openstack.org
       - csi.ovirt.org
       - csi.kubevirt.io
+      - csi.projected.shared.resources.openshift.io
```

Comment:
Is this the name we want? I'd think one of the following would be better:

1. `csi.projected-resources.openshift.io`
2. `csi.shared-resources.openshift.io`

Reply:
So none of the other names I hyphens, so I was not comfortable introducing that pattern, though I thought of it.  So unless @jsafrane says it is OK, I disagree there.

Of the two reduced versions you have, and losing hyphens, I like `csi.shared.resources.openshift.io` better.

You good with that @adambkaplan or are do you want to hold out for the hyphens being blessed?

Reply:
These are ultimately used as the name for a k8s CSIDriver object, which _at worst_ requires us to meet the DNS Label requirement [1]. Hyphens in the middle are OK.

Let's go with `csi.shared-resources.openshift.io`.

https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names

Reply:
since this is a product-facing name, does @smarterclayton need to sign off?

Reply:
updated @adambkaplan